src/app/app.routing.ts

```diff
@@ -131,6 +133,13 @@ const routes: Routes = [
                 component: CaseEditPageComponent,
               }
             ]
+          },
+          {
+            path: 'event/:eid',
```

Comment:
I find this path a bit confusing because we are not entering an event, but the case history associated to that event. 
Shall we change this to something like:
`event/:eid/history`

Reply:
yeah, history looks more logical, done.